external-builds/pytorch/build_prod_wheels.py

```diff
@@ -133,6 +137,16 @@ def get_rocm_sdk_version() -> str:
     raise ValueError(f"No rocm-sdk found in {' '.join(freeze_lines)}")
 
 
+def get_rocm_sdk_targets() -> str:
+    # Run `rocm-sdk targets` to get the default architecture
+    targets = capture([sys.executable, "-m", "rocm_sdk", "targets"], cwd=Path.cwd())
+    if not targets:
+        print("Warning: rocm-sdk targets returned empty or failed")
+        return ""
+    # Convert space-separated targets to comma-separated for PYTORCH_ROCM_ARCH
+    return targets.replace(" ", ",")
```

Comment:
I'm realizing I only tested that script with one target. But I thought it was semicolon delimited. If you've confirmed, that is fine/great. Let's just update the docs for that command in a followup.